certification/pyxis/layers.go

```diff
@@ -46,14 +44,13 @@ func (p *pyxisClient) CertifiedImagesContainingLayers(ctx context.Context, uncom
 	// make our query
 	httpClient, ok := p.Client.(*http.Client)
 	if !ok {
-		return nil, errors.ErrInvalidHttpClient
+		return nil, fmt.Errorf("client could not be used as http.Client")
 	}
 	client := graphql.NewClient(p.getPyxisGraphqlUrl(), httpClient).WithDebug(true)
 
 	err := client.Query(ctx, &query, variables)
 	if err != nil {
-		log.Error(fmt.Errorf("%w: %s", errors.ErrInvalidGraphqlQuery, err))
-		return nil, err
+		return nil, fmt.Errorf("could not marshal GraphQL query: %v", err)
```

Comment:
This `could not marshal` string could probably use some massaging. I know @acornett21 asked about this in another thread, but I'm thinking that this particular message isn't super useful to the user because it's internal to us (making a query to a remote, expecting to be able to receive an identifiable response).

Reply:
I don't know if there's a clear action I can recommend here.